mm/flow.py

```diff
@@ -37,11 +37,14 @@ def linear_segmentation_flow(feature_energy_net, alignment_cache=None):
     return net
 
 
-def alignment_flow(feature_net, alignment_cache_path=None):
+def alignment_flow(feature_net, alignment_cache_path=None, add_left_right_context_orth=False):
     assert "features" in feature_net.get_output_ports()
     net = FlowNetwork()
     net.add_output("alignments")
-    net.add_param(["id", "orthography", "TASK"])
+    if add_left_right_context_orth:
+        net.add_param(["id", "orthography", "left-context-orthography", "right-context-orthography", "TASK"])
+    else:
+        net.add_param(["id", "orthography", "TASK"])
```

Comment:
does the order matter? Otherwise I would suggest to factor out:
```suggestion
    net.add_param(["id", "orthography", "TASK"])
    if add_left_right_context_orth:
        net.add_param(["left-context-orthography", "right-context-orthography"])
```